packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -383,22 +416,22 @@ To check if a product is already on the wishlist pass it to `isInWishlist` metho
 
 ```vue
 <template>
-  // ...
+   <!-- ... -->
```

Comment:
here UI part is obsolete, this is not how a real wishlist works so there is no benefit of having this to copy

Reply:
removed